### PR TITLE
Changelog v0.11

### PR DESCRIPTION
**Pull request by jasonwilliams**
Changelog for v0.11

### CODE REVIEW

**Review by RageKnify — CHANGES_REQUESTED** [review]
Few things I think can be improved.

**Comment by jasonwilliams** [comment]
Thanks @RageKnify, merged

**Review by RageKnify — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,216 @@
...
+  Don't panic when function parameters share names (@AnnikaCodes)
+- [BUG #1024](https://github.com/boa-dev/boa/pull/1024):
+  Fix delete when the property is not configurable (@tofpie)
+- [BUG #1027](https://github.com/boa-dev/boa/pull/1027):
+  Supress regress errors on invalid escapes for regex (@jasonwilliams
+- [BUG #1031](https://github.com/boa-dev/boa/pull/1031):
+  Fixed some extra regex panics (@Razican)
+- [BUG 1049](https://github.com/boa-dev/boa/pull/1049):
+  Support overriding the `arguments` variable (@AnnikaCodes)
+
```
Comment (RageKnify) [inline CHANGELOG.md:144]: Add missing `#` and add line for #1050 that was merged in the mean time.
```suggestion
- [BUG #1049](https://github.com/boa-dev/boa/pull/1049):
  Support overriding the `arguments` variable (@AnnikaCodes)
- [BUG #1050](https://github.com/boa-dev/boa/pull/1050):
  Remove panic on named capture groups (@Razican)

```

**Comment by RageKnify** [comment]
Noticed that #1046 is also not in the list.

**Comment by tofpie** [comment]
Need to change also the version of the new crate `boa_unicode` introduced in #1003 
https://github.com/boa-dev/boa/blob/f66a0999e73d96718d55f24eeed1c35bfae2e592/boa_unicode/Cargo.toml#L1-L3

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,218 @@
...
+  Fix delete when the property is not configurable (@tofpie)
+- [BUG #1027](https://github.com/boa-dev/boa/pull/1027):
+  Supress regress errors on invalid escapes for regex (@jasonwilliams
+- [BUG #1031](https://github.com/boa-dev/boa/pull/1031):
+  Fixed some extra regex panics (@Razican)
+- [BUG #1049](https://github.com/boa-dev/boa/pull/1049):
+  Support overriding the `arguments` variable (@AnnikaCodes)
+- [BUG #1050](https://github.com/boa-dev/boa/pull/1050):
+  Remove panic on named capture groups (@Razican)
+
```
Comment (RageKnify) [inline CHANGELOG.md:146]:
```suggestion
- [BUG #1046](https://github.com/boa-dev/boa/pull/1046):
  Remove a few different panics (@Razican)
- [BUG #1051](https://github.com/boa-dev/boa/pull/1051):
  Fix parsing of arrow functions with 1 argument (@Lan2u)
- [BUG #1045](https://github.com/boa-dev/boa/pull/1045):
  Add newTarget to construct (@tofpie)
  
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,218 @@
...
+  Implement early errors for non-assignable nodes in assignment (@tofpie)
+- [FEATURE #1020](https://github.com/boa-dev/boa/pull/1020):
+  Implement Symbol.toPrimitive (@tofpie)
+- [FEATURE #976](https://github.com/boa-dev/boa/pull/976):
+  Implement for..in (@tofpie)
+- [FEATURE #1026](https://github.com/boa-dev/boa/pull/1026):
+  Implement String.prototype.split (@jevancc)
+- [FEATURE #1047](https://github.com/boa-dev/boa/pull/1047):
+  Added syntax highlighting for numbers, identifiers and template literals (@Razican)
+
```
Comment (RageKnify) [inline CHANGELOG.md:67]:
```suggestion
- [FEATURE #1003](https://github.com/boa-dev/boa/pull/1003):
  Improve Unicode support for identifier names (@jevancc)
  
```

**Review by RageKnify — COMMENTED** [review]
Proposals for the PRs that aren't mentioned, my classifications (feature vs bug vs internal) might be wrong though.

**Comment by Razican** [comment]
There are [3 more PRs](https://github.com/boa-dev/boa/pulls?q=is%3Aopen+is%3Apr+milestone%3Av0.11.0) ready to be merged that along with #997 we should add at least some of them.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,218 @@
...
+- [FEATURE #935](https://github.com/boa-dev/boa/pull/935):
+  Implement String.prototype.codePointAt (@devinus)
+- [FEATURE #961](https://github.com/boa-dev/boa/pull/961):
+  Implement the optional `space` parameter in `JSON.stringify` (@tofpie)
+- [FEATURE #962](https://github.com/boa-dev/boa/pull/962):
+  Implement Number.prototype.toPrecision (@NathanRoyer)
+- [FEATURE #983](https://github.com/boa-dev/boa/pull/983):
+  Implement Object.prototype.isPrototypeOf (@tofpie)
+- [FEATURE #659](https://github.com/boa-dev/boa/pull/659):
+  Error handling in environment (@54k1)
```
Comment (Razican) [inline CHANGELOG.md:48]: This would be a fix, since it fixed some panics.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,218 @@
...
+- [BUG #799](https://github.com/boa-dev/boa/pull/799):
+  Fix lexer span panic with carriage return (@vgel)
+- [BUG #812](https://github.com/boa-dev/boa/pull/812):
+  Fix 2 bugs that caused Test262 to fail (@RageKnify)
+- [BUG #826](https://github.com/boa-dev/boa/pull/826):
+  Fix tokenizing Unicode escape sequence in string literal (@HalidOdat)
+- [BUG #825](https://github.com/boa-dev/boa/pull/825):
+  calling "new" on a primitive value throw a type error (@dlemel8)
+- [BUG #853](https://github.com/boa-dev/boa/pull/853)
+  Handle invalid Unicode code point in the string literals(@jevancc)
```
Comment (Razican) [inline CHANGELOG.md:91]:
```suggestion
  Handle invalid Unicode code point in the string literals (@jevancc)
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,218 @@
...
+- [INTERNAL #837](https://github.com/boa-dev/boa/pull/837):
+  Set default-run to `boa` removing need for `--bin` (@RageKnify)
+- [INTERNAL #841](https://github.com/boa-dev/boa/pull/841):
+  Minor refactor and rename in eval() method (@croraf)
+- [INTERNAL #840](https://github.com/boa-dev/boa/pull/840):
+  fix(profiler): update profiler to match current measureme api (@neeldug)
+- [INTERNAL #838](https://github.com/boa-dev/boa/pull/838):
+  style(boa): minor cleanup (@neeldug)
+- [INTERNAL #869](https://github.com/boa-dev/boa/pull/869):
+  Updated cache in workflows
```
Comment (Razican) [inline CHANGELOG.md:182]:
```suggestion
  Updated cache in workflows (@Razican)
```

**Review by Razican — COMMENTED** [review]
Looks good, I found a couple of things that could be improved.

**Review by Lan2u — APPROVED** [review]
Apart from the comments already made looks good to me :)

**Pull request merged**